### ISSUE TITLE
Update Delay

### ISSUE THREAD

**Issue by digitaldavenyc** [richardwilly98/elasticsearch-river-mongodb#513]
How long does it take for the river plugin to update elasticsearch with data that has changed on mongo?

**Comment by dimon222** [comment]
Should be instantly I think. Depends on network and load on ES/MongoDB

**Comment by warm200** [comment]
it has a delay when I delete data on mongo, I start the server on local, and I use the fresh method of elasticsearch after I delete data on mongo,  it still has a delay, I have to refresh the page to see the change ,,don't know why,

**Comment by ewgRa** [comment]
It is not realtime. It has some delay, depends on network and so on. Also bear in mind that ES must index changes.

**Comment by warm200** [comment]
agree, delay is acceptable, maybe it's not a good idea that after you manipulate the data, you render them back using elasticsearch immediately,you should try to use the database side build-in functions to retrieve data after you do that, which has no delay
